PowerPointLabs/Test/FunctionalTest/ShapesLabTest.cs

```diff
@@ -87,5 +88,28 @@ private void DoubleClick(Control target)
             Point pt = target.PointToScreen(new Point(target.Width/2, target.Height/2));
             MouseUtil.SendMouseDoubleClick(pt.X, pt.Y);
         }
+
+        private void TestSaveShapesToShapesLabWithAddShapesButton(IShapesLabController shapesLab)
+        {
+            PpOperations.SelectSlide(6);
+            PpOperations.SelectShapesByPrefix("selectMeNow");
+            // Need to perform clicking of button in its own UI thread at ShapesLabController
+            // thus clicking cannot be performed directly in test script
```

Comment:
This is a good & valid point, but might work better in a README since it applies for all tests. Perhaps we can remove this for now.

Reply:
Ok yeah we could start a README somehow.